src/core/sprites/Sprite.js

```diff
@@ -522,7 +522,7 @@ Sprite.prototype.destroy = function (destroyTexture, destroyBaseTexture)
 
     if (destroyTexture)
     {
-        this._texture.destroy(destroyBaseTexture);
+        this._texture.destroy(destroyBaseTexture === undefined ? true : destroyBaseTexture);
```

Comment:
You are changing the default from false to true, so wondering why. Also should change the doc block along with this change.